File: java/coding/l1_e3.java
package coding;
import java.util.Scanner;

public class l1_e3{
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int n = scanner.nextInt();
        int area = scanner.nextInt();
        float iptu;

        if (n != 1 && n != 2 || area <= 0) {
            System.out.println("Entrada inválida!");
            scanner.close();
            return;
        }

        if (n == 1) {
            
            if(area < 200){
                iptu = area * 1.0f;
            } else {
                iptu = area * 1.2f;
            }

        } else {
            
            if (area < 200){
                iptu = area * 1.1f;
            } else {
                iptu = area * 1.3f;
            }

        }

        System.out.printf("%.2f\n", iptu);
        scanner.close();

    }
}

File: C++/l1_e3.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main () {
    
    int area;
    int escolha;

    cin >> escolha;
    cin >> area;

    if ((escolha != 1 && escolha != 2) || area <= 0) {
        cout << "Entrada inválida!" << endl;
        return 0;
    }

    if (escolha == 1) {
        if (area < 200) {
            cout << fixed << setprecision(2) << area * 1.0 << endl;
        } else {
            cout << fixed << setprecision(2) << area * 1.2 << endl;
        }
    } else if (escolha == 2) {
        if (area < 200) {
            cout << fixed << setprecision(2) << area * 1.1 << endl;
        } else {
            cout << fixed << setprecision(2) << area * 1.3 << endl;
        }
    }


    return 0;
}